designs/ble-mote.stanza

```diff
@@ -18,98 +18,92 @@ defpackage ocdb/designs/ble-mote :
-pcb-module pms7003 :
+pcb-bundle pms7003-interface :
   port vin : power
   port vin-mcu : power
   pin reset
   pin set
   port uart : uart()
+
+pcb-module pms7003 :
+  port interface : pms7003-interface
+  
   inst header : ocdb/amphenol/minitek127/component(10)
   place(header) at loc(0.0,0.0) on Top
-  net (vin.gnd vin-mcu.gnd header.p[3] header.p[4])
-  net (vin.vdd header.p[1] header.p[2])
-  net (uart.tx, header.p[10])
-  net (uart.rx, header.p[8])
-  net (reset, header.p[6])
-  net (set, header.p[9])
-  res-strap(vin-mcu.vdd, set, 10.0e3)
-  res-strap(vin-mcu.vdd, reset, 10.0e3)
+  net (interface.vin.gnd interface.vin-mcu.gnd header.p[3] header.p[4])
+  net (interface.vin.vdd header.p[1] header.p[2])
+  net (interface.uart.tx, header.p[10])
+  net (interface.uart.rx, header.p[8])
+  net (interface.reset, header.p[6])
+  net (interface.set, header.p[9])
+  res-strap(interface.vin-mcu.vdd, interface.set, 10.0e3)
+  res-strap(interface.vin-mcu.vdd, interface.reset, 10.0e3)
   property(header.p[1].power-pin) = PowerPin(min-typ-max(4.5, 5.0, 5.5))
   property(header.p[10].digital-output) = DigitalOutput(CMOSOutput(min-max(0.3, 0.5), min-max(2.6, 3.3)), false, header.p[1], header.p[3])
   property(header.p[8].digital-input) = DigitalInput(0.8, 2.6, header.p[1], header.p[3], 1.0e-6)
   property(header.p[6].digital-input) = DigitalInput(0.8, 2.6, header.p[1], header.p[3], 1.0e-6)
   property(header.p[9].digital-input) = DigitalInput(0.8, 2.6, header.p[1], header.p[3], 1.0e-6)
 
-pcb-module sensor-mote :
+pcb-module sensors :
+  port power-3v3 : power
+  port debug : uart([UART-DTR UART-RTS UART-RX UART-TX])
+  port particle-counter-interface : pms7003-interface
+  port humidity-i2c : i2c
+
   ; Add USB connector
-  public inst usb : micro-usb-connector
-  place(usb.con) at loc(-10.0, -7.3) on Top
+  public inst usb : usb2-on-a-usb-c-connector(ocdb/korean-hroparts-elec/TYPE-C-31-M-12/module(SinkDefault))
 
   ; Define ground net and assign symbol
   public net gnd (usb.usb-2.vbus.gnd)
   symbol(gnd) = ocdb/symbols/ground-sym
 
   ; Create a protected power and data interface for USB-2
-  val protected-usb = ocdb/protection/esd-clamp(usb.usb-2, gnd) as JITXObject
+  val protected-usb = ocdb/protection/esd-clamp(usb.usb-2, gnd)
 
   ; Specify a power regulator and name power nets
   public inst ldo : ocdb/diodes-incorporated/AP2112/module
   net (ldo.vin protected-usb.vbus)
   net P5V0 (ldo.vin.vdd ldo.en)
   net P3V3 (ldo.vout.vdd)
+  net VBUS (usb.usb-2.vbus.vdd)
   symbol(P5V0) = ocdb/symbols/supply-sym
   symbol(P3V3) = ocdb/symbols/supply-sym
 
-  ; Add a Bluetooth SoC
-  public inst proc : ocdb/espressif/ESP32-PICO-D4/module
-  public inst ant : inverted-f-antenna
-  net (proc.ant, ant.rf-in)
-  net (ant.gnd gnd)
-  place(ant) at loc(9.0, 5.0, -90.0) on Top
+  property(VBUS.net-voltage) = min-max(4.8, 5.2)
+  property(P5V0.net-voltage) = min-max(4.9, 5.1)
+  property(P3V3.net-voltage) = typ(3.3)
```

Comment:
I might suggest `min-typ-max(2.97, 3.30, 3.63)` for the specification of P3V3. Not a big deal, but it's more attuned to what users might need to check against.